examples/bri-3/src/bri/identity/bpiAccounts/models/bpiAccount.ts

```diff
@@ -18,21 +19,22 @@ export class BpiAccount {
   stateObjectProverSystem: string;
 
   @AutoMap()
-  stateObjectStorage: string;
+  stateTreeId: string;
```

Comment:
A BPI account has a direct line to a single state tree? also why needs id and tree itself?

Reply:
Currently yes, my understanding was that it is a bpi account that should store the state for a single workflow. Is that not the case? Regarding the id, i believe it is required because of the automapping setup which is not correct. We can fix this as part of the automapper fixes we ll do in mil5.